lib/Backend/JITType.cpp

```diff
@@ -104,26 +109,30 @@ JITTypeHolder::operator==(const JITTypeHolder& p) const
     return this->t == nullptr && p == nullptr;
 }
 
+template <class TAllocator>
 bool
-JITTypeHolder::operator!=(const JITTypeHolder& p) const
+JITTypeHolderBase<TAllocator>::operator!=(const JITTypeHolderBase& p) const
 {
     return !(*this == p);
 }
 
+template <class TAllocator>
 bool
-JITTypeHolder::operator==(const std::nullptr_t &p) const
+JITTypeHolderBase<TAllocator>::operator==(const std::nullptr_t &p) const
```

Comment:
>std::nullptr_t & [](start = 48, length = 16)

nit: the position (of space) of & is inconsistent between types JITTypeHolderBase and std::nullptr_t.